### DISCUSSION TITLE
Scoppy app - the future idea for allowing waveform arithmetic and boolean operations

### DISCUSSION THREAD

**Discussion by Athalbraht** [fhdm-dev/scoppy#102] [General]
Hi @fhdm-dev, I found your project some days ago, and after a few hours of fun with it, I think it would be great to see some kind of option allowing to display extra channel as a user-defined function of the others. It would be extremely useful, especially with the logic analyzer for educational applications. What do you think about this idea?

**Comment by fhdm-dev** [comment]
Hi,

I think it's a great idea if it makes the app more useful for educational purposes.

I already have plans to add separate math channel(s) for the oscilloscope mode but up until now I hadn't considered it for the logic analyzer mode.

Is this for a specific course you are teaching or a particular application?
